platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -753,7 +753,7 @@ public BuildOperationDescriptor.Builder description() {
                         projectPathString,
                         isVisible(),
                         isTransitive(),
-                        resolver.getRepositories()
+                        resolver.getAllRepositories()
```

Comment:
❓ I think this impacts what build scans shows.  If I'm following things correctly, I think this would show repositories that we would never look in.  Can you confirm that?

Reply:
It was always like that

Reply:
It would make sense to update this to the filtered list but for now since this is no behavior change I'd like to keep it as-is, as I think that change is more involved than it sounds

Reply:
ah you're right